### PR TITLE
Fixes For SafeStart

### PR DESCRIPTION
**Pull request by Njpatman**
- SafeStart Time is now sync'd to the server

- EH is now checked and re-applied periodically to make it compatible with Reforger Lobby

- Removed cap on players speaking range.

- Had the EHs apply to all players, so we removed any possibility of people accidentally shooting each other.

### CODE REVIEW

**Comment by Njpatman** [comment]
This is ready to be pushed through in its current state.

**Comment by tanaaKa** [comment]
Will review

File: scripts/Game/Systems/COA_SafeStart.c [hunk 1]
```diff
@@ -193,95 +184,107 @@ class CRF_TNK_SafestartComponent: SCR_BaseGameModeComponent
...
+		if (status) { // Turn on safestart
+			if (m_SafeStartEnabled) return;
 			
 			m_iTimeSafeStartBegan = GetGame().GetWorld().GetWorldTime();
 			m_SafeStartEnabled = true;
 			m_iSafeStartTimeRemaining = 35;
+			
 			GetGame().GetCallqueue().CallLater(CheckStartCountDown, 5000, true);
-			ToggleSafeStartEHs();
+			GetGame().GetCallqueue().CallLater(UpdateServerWorldTime, 250, true);
```
Comment (tanaaKa) [inline scripts/Game/Systems/COA_SafeStart.c:208]: Does this really need need to run every 250ms?

Comment (Njpatman) [reply scripts/Game/Systems/COA_SafeStart.c:208]: I'd prefer it that way to make the time display as smooth as possible, but it could run at 500 or 1000ms

Comment (tanaaKa) [reply scripts/Game/Systems/COA_SafeStart.c:208]: Yeah I mean, so far, enfusion has proved to be efficient, but it's good to ask the question. It's your call.

Comment (Njpatman) [reply scripts/Game/Systems/COA_SafeStart.c:208]: I like how it is right now; I will push it through.

**Pull request merged**